lookout/style/format/visualizer/server.py

```diff
@@ -42,20 +44,23 @@ def _rules_to_jsonable(rules: Rules, feature_extractor: FeatureExtractor
             for rule in rules.rules]
 
 
-def _vnode_to_dict(vnode: VirtualNode) -> Mapping[str, Any]:
-    return {
+def _vnode_to_jsonable(vnode: VirtualNode, labeled_indices: Mapping[int, int]
+                       ) -> Mapping[str, Any]:
+    jsonable = {
         "start": {"offset": int(vnode.start.offset),
                   "col": int(vnode.start.col),
                   "line": int(vnode.start.line)},
         "end": {"offset": int(vnode.end.offset),
                 "col": int(vnode.end.col),
                 "line": int(vnode.end.line)},
         "value": vnode.value,
-        "path": vnode.path,
-        "roles": [role for role in vnode.node.roles] if vnode.node else [],
-        "y": vnode.y,
-        "internal_type": vnode.node.internal_type if vnode.node else None,
+        "path": vnode.path
```

Comment:
```suggestion
        "path": vnode.path,
```

But as you wish

Reply:
I agree. We should probably add that to flake8. I'll create an issue